dockstore-webservice/src/main/java/io/dockstore/webservice/api/Config.java

```diff
@@ -0,0 +1,58 @@
+package io.dockstore.webservice.api;
+
+import java.lang.reflect.InvocationTargetException;
+
+import io.dockstore.webservice.DockstoreWebserviceConfiguration;
+import io.swagger.annotations.ApiModel;
+import org.apache.commons.beanutils.BeanUtils;
+
+@ApiModel("Config")
```

Comment:
Seems a bit generic

Reply:
Agreed, but I'm having a hard time thinking of anything better. Any ideas?

UIConfig? Configuration? UIConfiguration? Those aren't really any better, but I'm drawing a blank.

Reply:
I kept the name but added a description.